library/n2_protocol.hpp

```diff
@@ -147,4 +147,36 @@ struct remove_request : n2_body {
 	               dnet_time timestamp);
 };
 
+struct file_pointer {
+	int fd = -1;
+	uint64_t offset = 0;
+	uint64_t size = 0;
+	int on_exit = 0;
+
+	file_pointer() = default;
```

Comment:
why do we need such empty ctor?

Reply:
For using it in `chunk_t::chunk_t(const data_pointer &data);`